_test_/users.test.js

```diff
@@ -0,0 +1,13 @@
+const supertest = require('supertest');
+const endpoint = 'https://rest-fetch.herokuapp.com/users'
+
+describe("Testing the users API", () => {
+
+    it("Tests the users route and returns true for status", async () => {
+
+        const response = await supertest(endpoint).get('/');
+        expect(response.status).toBe(200);
```

Comment:
This is just the partial test of the code. You need to check for the response body not just the 200 OK status. And I can see the test for just a single route.